tests/test_accelerator.py

```diff
@@ -326,12 +326,17 @@ def test_accelerator_bnb_cpu_error(self):
 
     @slow
     @require_bnb
-    @require_multi_gpu
-    def test_accelerator_bnb_multi_gpu(self):
+    @require_multi_device
+    def test_accelerator_bnb_multi_device(self):
         """Tests that the accelerator can be used with the BNB library."""
         from transformers import AutoModelForCausalLM
 
-        PartialState._shared_state = {"distributed_type": DistributedType.MULTI_GPU}
+        if torch_device == "cuda":
```

Comment:
This is fine for now, if we notice needing this more then we should define this somewhere before as a shared var similar to `torch_device` (but inside the test file itself)

Reply:
That makes sense. I will try if need more later.